CONTRIBUTING.md

```diff
@@ -1,68 +1,33 @@
-if the problem has already been reported (or fixed in a master branch).
-
-If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/packit/ogr/issues/new).
-Be sure to include a **descriptive title and a clear description**. Ideally, please provide:
-
-- version of ogr you are using (`rpm -q python3-ogr` or `pip3 freeze | grep ogr`)
-- the command you executed and a debug output (using option `--debug`)
-
-If possible, add a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
-
-**Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
-You can also comment on the closed issue to indicate that upstream should provide a new release with a fix.
-
-### Suggesting Enhancements
-
-Enhancement suggestions are tracked as GitHub issues.
-When you are creating an enhancement issue, **use a clear and descriptive title** and **provide a clear description of the suggested enhancement** in as many details as possible.
+- [List of known issues](https://github.com/packit/ogr/issues)
+- Getting a version of `ogr`:<br>
+  `rpm -q python3-ogr` or `pip3 freeze | grep ogr`
+- [In case you need to create a new issue](https://github.com/packit/ogr/issues/new)
 
-## Guidelines for Developers
-
-If you would like to contribute code to the `ogr` project, this section is for you!
-
-### Is this your first contribution?
-
-Please take a few minutes to read GitHub's guide on [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/).
-It's a quick read, and it's a great way to introduce yourself to how things work behind the scenes in open-source projects.
-
-### Dependencies
-
-If you are introducing a new dependency, please make sure it's added to:
-
-- [setup.cfg](setup.cfg)
-
-### Documentation
+## Documentation
 
 If you want to update documentation, [README.md](README.md) is the file you're looking for.
 
-#### Changelog
+## Links to documentation
 
-When you are contributing to changelog, please follow these suggestions:
+Here are some links to documentation that could be helpful when contributing:
 
-- Changelog is maintained mainly during release process.
-- The changelog is meant to be read by everyone. Imagine that an average user
-  will read it and should understand the changes.
-- Every line should be a complete sentence. Either tell what is the change that the tool is doing or describe it precisely:
-  - Bad: `Use search method in label regex`
-  - Good: `Ogr now uses search method when...`
-- And finally, with the changelogs we are essentially selling our projects:
-  think about a situation that you met someone at a conference and you are
-  trying to convince the person to use the project and that the changelog
-  should help with that.
+- GitHub
```

Comment:
What about making it clear that we use `PyGithub` for implementing GitHub, `python-gitlab` for GitLab and directly `requests` for Pagure?

I know It's there but let's make it really clear for newcomers. Thanks!